modules/install-consul/install-consul

```diff
@@ -77,17 +77,33 @@ function install_supervisord_debian {
   sudo systemctl enable supervisor
 }
 
+function two_way_symlink() {
+  src=$1
+  dest=$2
```

Comment:
We follow the [google shell style guide](https://google.github.io/styleguide/shell.xml), so all function params should be `local readonly` and wrap the argument in double quotes (e.g., `"$1"`).